templates/car_input.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <h1>Welcome to the Automotive Solution Tool!</h1>
+</head>
+<body>
+
+    <p>Please fill out the boxes below to search for your desired car:</p>
```

Comment:
`<p>` is for paragraph, we can use this to store all of the welcome details, for now - replace with an `<h3>` heading